packages/admin-e2e-tests/src/specs/analytics/analytics-overview.ts

```diff
@@ -11,44 +11,45 @@ const testAdminAnalyticsOverview = () => {
 	describe( 'Analytics pages', () => {
 		const analyticsPage = new AnalyticsOverview( page );
 		const login = new Login( page );
+		const sectionTitles = [ 'Performance', 'Charts', 'Leaderboards' ];
+		const titlesString = sectionTitles.join( ', ' );
 
 		beforeAll( async () => {
 			await login.login();
 			await analyticsPage.navigate();
 			await analyticsPage.isDisplayed();
+			// Restore original order to sections
+			for ( let t = 0; t < sectionTitles.length; t++ ) {
+				const visibleSections = await analyticsPage.getSectionTitles();
+				if ( visibleSections.indexOf( sectionTitles[ t ] ) < 0 ) {
+					await analyticsPage.addSection( sectionTitles[ t ] );
```

Comment:
I think this would be ok for now, although we should probably replace it with clearing the `dashboard_sections` in the users `woocommerce_meta` settings. I guess this could also be part of the reset functionality, I added this to the list of todo's [for the reset PR](https://github.com/woocommerce/woocommerce-admin/pull/7064#issue-651907939)

Reply:
@louwie17 That sounds good. I didn't investigate further but resetting the user meta settings may be possible with the rest api - https://developer.wordpress.org/rest-api/reference/users/

Reply:
Yeah that's what I was thinking, and we can make use of the `HTTPClientFactory` for that. This can be merged before that though.